src/QsCompiler/SyntaxProcessor/ScopeTools.fs

```diff
@@ -226,9 +233,11 @@ type SymbolTracker(globals: NamespaceManager, sourceFile, parent: QsQualifiedNam
             LocalVariableDeclaration<_>.New mut ((decl.Position, decl.Range), name, decl.Type, qDep)
             |> this.TryAddVariableDeclartion
 
+    /// <summary>
     /// Updates the quantum dependency of the local variable with the given name.
-    /// Throws an ArgumentException if no local variable with the given name is visible on the current scope.
-    /// Throws an InvalidOperationException if no scope is currently open, or the variable is immutable.
+    /// </summary>
+    /// <exception cref="ArgumentException"><paramref name="varName"/> is not visible in the current scope.</exception>
+    /// <exception cref="InvalidOperationException">No scope is currently open, or <paramref name="varName"/> is immutable.</exception>
```

Comment:
I think this might also work better as two separate tags:

```suggestion
    /// <exception cref="InvalidOperationException">No scope is currently open.
    /// <exception cref="InvalidOperationException"><paramref name="varName"/> is immutable.</exception>
```